.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules
-public/assets/css/style.css*
 public/OpenFinD3FC.zip
-.idea
\ No newline at end of file
+.idea
+public/assets/fonts
+public/assets/css
+public/app.json
```

Comment:
That will ignore all the files. I hope to have a directory structure there once the app gets bigger?